apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/dao/ShardDao.java

```diff
@@ -30,6 +30,10 @@
     @SqlQuery("SELECT count(*) FROM shard")
     long countShard();
 
+    @Transactional(readOnly = true)
+    @SqlQuery("SELECT IFNULL(max(SHARD_ID) + 1, 1) FROM shard")
```

Comment:
Sequence doesn't suite here, because we initialize shard db two times with two composed 'schema classes'. So we want to get the same number in two cases, or util we put 'record #1'  into table. Sequence will return two values (like 1, 2) in our case and that it not quite correct for our case.